package.json

```diff
@@ -69,17 +75,9 @@
     "typechain": "^8.0.0",
     "typescript": "^4.6.3"
   },
-  "dependencies": {
-    "@aave/aave-token": "^1.0.4",
-    "@aave/core-v3": "^1.17.2",
-    "@aave/deploy-v3": "^1.55.3",
-    "@aave/periphery-v3": "^2.0.1",
-    "@aave/safety-module": "^1.0.3",
-    "@openzeppelin/contracts": "^4.5.0"
-  },
   "overrides": {
     "@nomicfoundation/hardhat-toolbox": {
-      "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers"
+      "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@0.3.0-beta.13"
```

Comment:
why this change is necessary? would prefer to avoid "beta" packages

Reply:
Let's keep it as it is, given that there are [compatibility issues](https://github.com/wighawag/hardhat-deploy-ethers/issues/27#issuecomment-1431411121) between hardhat-deploy and hardhat-toolbox..,